Build/GlobalFacetManifest.NetFramework.xml

```diff
@@ -43,14 +43,39 @@
            feature="SharpZipLib"
            layer="Core"/>
 
+  <!-- SharpZipLib -->
```

Comment:
Why are those changes necessary? They were not in the original PR and seem out of place.